NetKVM/Common/ParaNdis_Util.cpp

```diff
@@ -130,12 +126,15 @@ bool CSystemThread::Start(PVOID Context)
     m_Context = Context;
     UpdateTimestamp(m_StartTime);
     NTSTATUS status = PsCreateSystemThread(
-        &m_hThread, GENERIC_READ, NULL, NULL, NULL,
-        [](PVOID Ctx)
-        {
-            ((CSystemThread*)Ctx)->ThreadProc();
-        },
-        this);
+                                                                                                        &m_hThread,
+                                                                                                        GENERIC_READ,
+                                                                                                        NULL,
+                                                                                                        NULL,
+                                                                                                        NULL,
```

Comment:
Try to fix manually by putting "NTSTATUS status = PsCreateSystemThread( &m_hThread," in the same line